tests/ff_module-progress.test.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+var React = require('react');
+require('./lib/utils').bootstrapBrowser();
+var TestUtils = require('react-addons-test-utils');
+var expect = require('chai').expect;
+
+var Progress = require('../blocks/core/ff_module/ff_module-progress/ff_module-progress.js');
+
+describe('Progress', function() {
+
+  it('should always render a progress bar', function() {
+    var element = React.createElement(Progress, {})
+		var component = TestUtils.renderIntoDocument(element);
+		expect(component).to.exist;
+  })
+
+  it('should always render a progress bar from minimal data', function() {
```

Comment:
You should probably also test that the bar is rendering expected values from provided data, like x marked is x% wide (you could add a set-width wrapper if the width is percentage based)